client/locales/en.yml

```
@@ -306,6 +324,25 @@ forms:
       password: Password
       remember_me: Remember me
       username: Username
+  register:
+    errors:
+      missing_username: Username is required
+      invalid_username: Invalid username format
+      username_too_short: The username must be at least {min} characters long
+      username_too_long: The username must be at most {max} characters long
+      missing_email: Email address is required
+      invalid_email: Invalid email address format
+      missing_password: Password is required
+      password_too_short: The password must be at least {min} characters long
+      password_too_long: The password must be at most {max} characters long
+      missing_password_confirmation: Password confirmation is required
+      passwords_mismatch: The passwords do not match
+    fields:
+      username: Username
+      email: E-mail address
```

We have been consistent with "email" in english, so we should stick to it, or change it everywhere

```suggestion
      email: Email address
```